modules/ROOT/pages/dw-core-functions-orderby.adoc

```diff
@@ -62,13 +62,13 @@ orderByLetter: [{ letter: "d" }, { letter: "e" }, { letter: "c" }, { letter: "a"
 }
 ----
 
-== Example
+=== Example
 
 The `orderBy` function does not have an option to order in descending order
 instead of ascending. In these cases, you can simply invert the order of
 the resulting array using `-`, for example:
```

Comment:
Delete 2nd sentence (it's not true). Just say "to invert the order from ascending to descending, use`-`. For example: